storage/innobase/xtrabackup/test/inc/common.sh

```diff
@@ -1150,5 +1150,33 @@ function backup_and_restore() {
   verify_db_state $db
 }
 
+### kill all running queries accept the condition argument
+## example kill_condition_queries "time > 0 & INFO like '%SLEEP%'"
+function kill_condition_queries()
+{
+  local condition=$1
+  run_cmd $MYSQL $MYSQL_ARGS --force --batch  test <<EOF
+  select concat('KILL ',id,';') from information_schema.processlist
+  where $condition into outfile '$MYSQLD_TMPDIR/killcondition.sql';
+  source $MYSQLD_TMPDIR/killcondition.sql;
+EOF
+ rm -f $MYSQLD_TMPDIR/killcondition.sql
```

Comment:
EOF and rm -rf are wrongly aligned

Reply:
EOF should come without any spaces. rm -rf is aligned to 2 spaces it it inside function.